src/sign.js

```diff
@@ -0,0 +1,88 @@
+import SIGN from './constants/VALIDATION.js';
+
+const emailInput = document.getElementById('email');
+const emailError = document.getElementById('email-error');
+const pwInput = document.getElementById('password');
+const pwError = document.getElementById('password-error');
+const pwToggle = document.querySelector('.password-eye');
+const signBtn = document.querySelector('.sign-normal__btn');
+
+const applyErrorStyle = (el) => {
+  el.style.border = '1px solid var(--red)';
+};
+const resetErrorStyle = (el) => {
+  el.style.border = '';
+};
+
+const applyError = (errorEl, errorMsg, input) => {
+  errorEl.innerText = errorMsg;
+  applyErrorStyle(input);
+};
+const resetError = (errorEl, input) => {
+  errorEl.innerText = '';
+  resetErrorStyle(input);
+};
+
+const isEmailFormat = (email) => {
+  return SIGN.EMAIL_REGEX.test(email);
+};
+const isPwFormat = (pw) => {
+  return SIGN.PW_REGEX.test(pw);
+};
```

Comment:
> 5. 마지막으로 2번 질문과 비슷한데 전체적인 코드를 봤을 때 더 분리하거나 아니면 합치면 좋을 부분이 있을까요? ex) 저는 아래와 같은 부분들이 고민되었던 것 같습니다.

합치는 것을 고민하셨군요~ 사진 속에서 작성해주신 것과 같이 합치는 것도 좋은 방법인데요. 조금만 더 다듬어본다면 아래와 같이 isValidFormat에서 Regex도 같이 가져가면 좋을 것 같습니다. action 값도 상수로 같이 관리하면 더 좋겠네요!

```suggestion
const isValidFormat = (action, input) => {
  if(action === "PW") return SIGN.PW_REGEX.test(input);
  if(action === "EMAIL") return SIGN.EMAIL_REGEX.test(input);
};
```

Reply:
오웅 네네!!  action도 상수로 가져가도록 하겠습니다.